src/component/ChangePassword/ChangePassword.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import Button from "react-bootstrap/Button";
+import Modal from "react-bootstrap/Modal";
+import Form from "react-bootstrap/Form";
+import { ApiClient } from "../../request/request";
+import { loginUser } from "../../utils/apiRequest";
+
+import { useNavigate } from "react-router-dom";
+import { useDispatch } from "react-redux";
+
+import styles from "./ChangePassword.module.scss";
+import classNames from "classnames/bind";
+const cx = classNames.bind(styles);
+
+function ChangePassword() {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const [show, setShow] = useState(false);
+  const [usename, setusername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  const handleChangePassword = async (e) => {
+    e.preventDefault();
+    const user = {
+      username: usename,
+      newPassword: password,
+    };
+    const newUser = {
+      username: usename,
+      password: password,
+    };
+    await ApiClient.patch(`/api/users/${localStorage.getItem("id")}`, user)
+      .then((res) => {
+        if (res.status === 200) {
+          alert("Success");
+          handleClose();
+          loginUser(newUser, dispatch, navigate);
```

Comment:
chị nghĩ chỗ này nên viết thẳng như này luôn, không cần tạo biến newUser nữa, như vậy sẽ gọn hơn
```suggestion
          loginUser({ username, password }, dispatch, navigate);
```